### ISSUE TITLE
March swarm : Kibana index pattern not configured

### ISSUE THREAD

**Issue by gene1wood** [mozilla/MozDef#1174]
![image](https://user-images.githubusercontent.com/1134034/54869453-b5d98080-4d55-11e9-87d1-c0cd944357dd.png)

```
 Configure an index pattern

In order to use Kibana you must configure at least one index pattern. Index patterns are used to identify the Elasticsearch index to run search and analytics against. They are also used to configure fields. 
```

When launching a new deployment of MozDef in AWS, kibana has no index pattern to start.

@pwnbus remind me where the code was that you'd added that should do this and I'll look into why it was working (to setup an initial index pattern) and isn't now for me.

**Comment by gene1wood** [comment]
pwnbus says the area to look at is 

Hmm, the bootstrap container is the thing that should be creating those index patterns in kibana, specifically this is the script https://github.com/mozilla/MozDef/blob/master/docker/compose/mozdef_bootstrap/files/initial_setup.py

**Comment by gene1wood** [comment]
@pwnbus it looks like it is creating index but not the `events` index?

```

**Comment by Phrozyn** [comment]
it's creating it, remember it gets created as events-YYYYMMDD

The alias is also created, it's just not being set as default, even though we do attempt to do that in the code here:
https://github.com/mozilla/MozDef/blob/master/docker/compose/mozdef_bootstrap/files/initial_setup.py#L141

if you look at your image you posted, right under the yellow warning are the index patterns it has.
events is there. So something is causing it to simply not get selected as the default.

**Comment by Phrozyn** [comment]
Looking at that code, it appears to be searching for hits: 0
however, when i do a query using dev tools, the entry comes back as "hits": { "total": 0}
Testing a fix for this now.

**Comment by gene1wood** [comment]
Fixed in #1184 

Root cause was the code previously used a hard coded kibana version (which did not match the version in AWS ES)
